tests/functional/behave_features/HC-06_provider_delivery_control.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Report only submission with provider control settings
+  Partners can prevent publication of their chart by submitting
+  error-free report that was generated by chart-verifier and with
+  prvider controlled delivery set in the report and the OWNERS file.
+
+  @external-feedback @partners @smoke
+  Scenario Outline: [HC-06-001] A partner associate submits an error-free report with provider controlled delivery
+    Given the vendor "<vendor>" has a valid identity as "<vendor_type>"
+    And provider delivery control is set to "<provider_control_owners>" in the OWNERS file
+    And a "<report_path>" is provided
+    And provider delivery control is set to "<provider_control_report>" in the report
+    When the user sends a pull request with the report
+    Then the user sees the pull request is merged
+    And the "<index_file>" is updated with an entry for the submitted chart
+
+    Examples:
+      | vendor_type  | vendor    | report_path               | index_file                        | provider_control_owners | provider_control_report |
+      | partners     | hashicorp | tests/data/report.yaml    | unpublished-certified-charts.yaml | true                    | true                    |
+
+  @external-feedback @partners
```

Comment:
@external-feedback is the tag used for identifying test cases which we have added as part of customer feedback. Earlier QE had a initiative to calculate percentage of tests which are being as part of customer feedback in the test suite. It is part of that. I think since the tag is applicable to all the examples of the scenario, so tagged at the scenario level to avoid tagging at each example. @mmulholla WDYT?

Reply:
Would rather see the @partner tag on the examples to be consistent with other tests. 
@external-feedback is fine but need to review tests to see which should have the tag.

Reply:
moved the `partners` tag to the example level to be consistent with other tests. @mmulholla since you are more closely working with chart-verifier, would require your help to identify what other tests we can mark as `@external-feedback`